tests/cloudformation/checks/resource/aws/test_DynamodbGlobalTableRecovery.py

```diff
@@ -0,0 +1,26 @@
+import os
+import unittest
+
+from checkov.cloudformation.checks.resource.aws.DynamodbGlobalTableRecovery import check
+from checkov.cloudformation.runner import Runner
+from checkov.runner_filter import RunnerFilter
+
+
+class TestDynamodbGlobalTableRecovery(unittest.TestCase):
+
+    def test_summary(self):
+        runner = Runner()
+        current_dir = os.path.dirname(os.path.realpath(__file__))
+
+        test_files_dir = current_dir + "/example_DynamodbGlobalTableRecovery"
+        report = runner.run(root_folder=test_files_dir,runner_filter=RunnerFilter(checks=[check.id]))
+        summary = report.get_summary()
+
+        self.assertEqual(summary['passed'], 1)
+        self.assertEqual(summary['failed'], 2)
+        self.assertEqual(summary['skipped'], 0)
+        self.assertEqual(summary['parsing_errors'], 0)
+
```

Comment:
can you also check for which resource exactly fails, like here
https://github.com/bridgecrewio/checkov/blob/4fa8fbb2bfa7931a01e9da3fc1c46d1e49d18d5d/tests/cloudformation/checks/resource/aws/test_RDSMultiAZEnabled.py#L19-L36
I think it would be also great, if you give them a proper name too 😄

Reply:
@gruebel I have fixed the cfn-lint issue.

Reply:
this comment has nothing to do with the cfn-lint errors. It would be great if you can check for the exact passed & failed resources, like in the example I added.

Reply:
@gruebel Yes as per your suggestion I have modified the test case for passed and failed resources.

Reply:
@gruebel Yes as per your suggestion I have modified the test case for passed and failed resources.